pkgs/love2d/example/assets/love_example_browser/conf.lua

```
@@ -0,0 +1,5 @@
+function love.conf(t)
```

Luacheck (1.2.0)

[warning] unused argument 't'

(W212)